docfx_project/articles/bootstrapping-trust/part-04.certificate-authorities-and-chains-of-trust.md

```diff
@@ -0,0 +1,181 @@
+# Bootstrapping Trust
+
+## Part 4: Certificate Authorities & Chains of Trust
+
+If you have read through the entire series up to here, welcome! If you
+have not, please consider reading the whole series:
+
+- [Part 1: Encryption Everywhere](./part-01.encryption-everywhere.md)
+- [Part 2: A Primer On Public-Key Cryptography](./part-02.a-primer-on-public-key-cryptography.md)
+- [Part 3: Certificates](./part-03.certificates.md)
+- [Part 4: Certificate Authorities & Chains Of Trust](./part-04.certificate-authorities-and-chains-of-trust.md)
+- [Part 5: Bootstrapping Trust](./part-05.bootstrapping-trust.md)
+
+
+### Root & Intermediate Certificate Authorities (CAs)
+
+Not all certificates are the same! Certificates have different
+capabilities depending on their usage attributes and extensions. The
+previous article in this series mentioned a few of those attributes and
+extensions. Two of those were `clientAuth,` for client certificates, and
+`serverAuth,` for server certificates, which play an essential role in
```

Comment:
clientAuth, and serverAuth, should have the comma outside the ticks :)

Reply:
Fixed